app/views/content/teaching-internship-providers.md

```diff
@@ -158,32 +183,39 @@ provider_groups:
       name: Katie Cornforth
       email: k.cornforth@xaviercet.org.uk
       subjects: chemistry, computing, maths, physics, languages
+  'London  ':
```

Comment:
@ethax-ross One of the rows in the spreadsheet has trailing spaces after London so it has uploaded as a separate region

I will flag this to the team but is it quickest to fix ourselves so we can get this into prod on Friday morning or do you want a new spreadsheet to upload?